pcdsdevices/sim.py

```diff
@@ -64,6 +64,12 @@ def _setup_move(self, position, status):
 
         def update_thread(positioner, goal):
             positioner._moving = True
+            if positioner.position == goal:
+                # if already at the requested position, mark success
+                positioner._set_position(goal)  # needed for LiveTable format
```

Comment:
I'm not sure why this line is needed.  Without it, LiveTable fails to write new rows after the first.

Reply:
Does `_set_position` do a subscription callback by chance?